### PR TITLE
account_objects of new RPC framework

### PR DESCRIPTION
**Pull request by cindyyan317**

### CODE REVIEW

File: src/rpc/ngHandlers/AccountObjects.cpp [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+    return response;
+}
+
+void
+tag_invoke(boost::json::value_from_tag, boost::json::value& jv, AccountObjectsHandler::Output const& output)
+{
+    boost::json::array objects;
+    for (auto const& sle : output.accountObjects)
+    {
```
Comment (godexsoft) [inline src/rpc/ngHandlers/AccountObjects.cpp]: Could remove the extra braces

File: src/rpc/ngHandlers/AccountObjects.cpp [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        objects.push_back(RPC::toJson(sle));
+    }
```
Comment (godexsoft) [inline src/rpc/ngHandlers/AccountObjects.cpp]: And add a space after the for loop

File: src/rpc/ngHandlers/AccountObjects.cpp [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    jv = {
+        {JS(ledger_hash), output.ledgerHash},
+        {JS(ledger_index), output.ledgerIndex},
+        {JS(validated), output.validated},
+        {JS(limit), output.limit},
+        {JS(account), output.account},
+        {JS(account_objects), objects}};
```
Comment (godexsoft) [inline src/rpc/ngHandlers/AccountObjects.cpp:94]: Line of space would be nice before the if

File: src/rpc/ngHandlers/AccountObjects.cpp [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+    if (output.marker)
+        jv.as_object()[JS(marker)] = *(output.marker);
+}
+
+AccountObjectsHandler::Input
+tag_invoke(boost::json::value_to_tag<AccountObjectsHandler::Input>, boost::json::value const& jv)
+{
+    auto const& jsonObject = jv.as_object();
+    AccountObjectsHandler::Input input;
+    input.account = jv.at(JS(account)).as_string().c_str();
```
Comment (godexsoft) [inline src/rpc/ngHandlers/AccountObjects.cpp:105]: I often ignore these small things but really we should put more spacing before/after `if`, `for` etc.

File: src/rpc/ngHandlers/AccountObjects.h [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+    {
+        static auto const rpcSpec = RpcSpec{
+            {JS(account), validation::Required{}, validation::AccountValidator},
+            {JS(ledger_hash), validation::Uint256HexStringValidator},
+            {JS(ledger_index), validation::LedgerIndexValidator},
+            {JS(limit), validation::Type<uint32_t>{}, validation::Between(10, 400)},
+            {JS(type),
+             validation::Type<std::string>{},
+             validation::OneOf<std::string>{
+                 "state",
```
Comment (godexsoft) [inline src/rpc/ngHandlers/AccountObjects.h:77]: There are no JS() for any of these?

File: src/rpc/ngHandlers/AccountObjects.h [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    process(Input input, Context const& ctx) const;
+
+private:
+    friend void
+    tag_invoke(boost::json::value_from_tag, boost::json::value& jv, Output const& output);
+
+    friend Input
+    tag_invoke(boost::json::value_to_tag<Input>, boost::json::value const& jv);
+
+    static std::unordered_map<std::string, ripple::LedgerEntryType> const TYPESMAP;
```
Comment (godexsoft) [inline src/rpc/ngHandlers/AccountObjects.h]: This would probably be better placed at the top private section, next to other 'data'

**Review by godexsoft — CHANGES_REQUESTED** [review]
Adding a few small suggestions - overall looking good!

File: src/rpc/ngHandlers/AccountObjects.h (hunk 5 above)
Comment (cindyyan317) [reply src/rpc/ngHandlers/AccountObjects.h:77]: No~ unfortunately

**Review by godexsoft — APPROVED** [review]
LGTM!

**Pull request merged**